src/planner/create_plan.cpp

```diff
@@ -180,25 +191,38 @@ CreatePlan::CreatePlan(parser::CreateStatement *parse_tree)
 
       break;
     }
+    case parser::CreateStatement::CreateType::kSequence: {
+      create_type = CreateType::SEQUENCE;
+      database_name = std::string(parse_tree->GetDatabaseName());
+
+      sequence_name = parse_tree->sequence_name;
```

Comment:
Do you still need to assign default values for fields in createStatement if you always assign values to them explictly here?

Reply:
The value could be empty in the postgres, which leave some field undefined.